apps/phone/src/apps/settings/components/SettingsApp.tsx

```diff
@@ -163,7 +163,7 @@ export const SettingsApp: React.FC = () => {
 
   const [openMenu, closeMenu, ContextMenu, isMenuOpen] = useContextMenu();
   const classes = useStyles();
-
+  const theme = useTheme();
```

Comment:
Why passing theme as prop instead of using theme when needed?

Reply:
Except, I am using it when needed....
Anywhere i put it, it's required for the phone to work correctly with themes